### ISSUE TITLE
Ability to specify a whitelist

### ISSUE THREAD

**Issue by simonexmachina** [quaertym/ember-cli-dependency-checker#14]
It's possible (and not uncommon) to install packages using bower that don't have a bower.json. For example, Sinon.js doesn't provide a Bower package, so you install it using an HTTP URL:

```
"sinon": "http://sinonjs.org/releases/sinon-1.10.0.js",
```

This is reported as "(not installed)" by this module and prevents ember-cli from starting. The only workaround that I can see is to remove this module.

It would be good to be able to specify a "whitelist" of packages that should be ignored.
